acceptancetests/assess_caas_bootstrap.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+""" Test caas k8s cluster bootstrap
+"""
```

Comment:
This should give a brief outline what it's actually asserting, "Spinning up a k8s cluster and ensuring that x is correct."

Reply:
agreed